udata_front/theme/gouvfr/datagouv-components/src/components/DatasetCard/DatasetCard.stories.ts

```diff
@@ -28,7 +28,7 @@ const datasetRoutes = [
     path: '/organizations/:id',
     component: TestComponent
   }
-]
+];
 
 const dataset: DatasetV2 = {
```

Comment:
Can you compute dates based on the current one instead for stories showing `X day ago` ? Otherwise Chromatic will detect a visual change each day.